api/optimeet/groups/views.py

```diff
@@ -63,6 +63,29 @@ def add_users_to_group(request, group_id):
     
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+@csrf_exempt
+@api_view(['GET'])  
+def votes(request, group_id):
+        votes = models.Votes.objects.filter(group_id=group_id)
+        serializer = serializers.VotesSerializer(votes, many=True)
+        return Response(serializer.data)
+    
+@csrf_exempt
+@api_view(['POST'])
+def create_vote(request, group_id):
+
+    try:
+        group = models.Group.objects.get(group_id=group_id)
+    except models.Group.DoesNotExist:
+        raise Http404  
+    
+    request.data['group_id'] = group_id
+    serializer = serializers.VotesSerializer(data=request.data, context={'group_id': group_id})
```

Comment:
Instead of setting `group_id` into the request, which could be problematic before serializing, let's add it in the `create` function in the serializer like we did last time with UserGroupSerializer. You're already passing the group_id to the context, so you just need to use it in a new `create` function.